blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -279,8 +280,6 @@ function createLI(li) {
 
 async function buildProductCard(program) {
     const productMapping = await getProductMapping(program.productOffering);
-    if (!productMapping.label)
```

Comment:
@mdickson-adbe this fix Tyrone added to show Not Available if product is empty for a program
![image](https://github.com/hlxsites/adobe-gmo/assets/61603050/f9098928-a885-4e31-adb7-34c948a7a02f)

Reply:
I have a function in shared-program.js that should do the same.. let me push a fix.

Reply:
![image](https://github.com/hlxsites/adobe-gmo/assets/95774602/70e9212f-ade5-4235-b340-31b707c0ea80)